lib/Fmt.ml

```diff
@@ -88,30 +88,36 @@ let with_box_debug k = with_pp (Box_debug.with_box (fun fs -> eval fs k))
 (** Break hints and format strings --------------------------------------*)
 
 let break n o =
+  let stack = Box_debug.get_stack () in
   with_pp (fun fs ->
-      Box_debug.break fs n o ;
+      Box_debug.break fs n o ~stack ;
       Format_.pp_print_break fs n o )
 
 let force_break = break 1000 0
 
 let space_break =
+  (* a stack is useless here, this would require adding a unit parameter *)
+  let stack = "" in
   with_pp (fun fs ->
-      Box_debug.space_break fs ;
+      Box_debug.space_break ~stack fs ;
       Format_.pp_print_space fs () )
 
 let cut_break =
+  let stack = Box_debug.get_stack () in
```

Comment:
Should be the same as in the space case, no trace ? Same in `force_newline`.

Reply:
Yes this is true

Reply:
There's still the issue in `force_newline`.